examples/getting_started/2_Running_quantum_circuits_on_QPU_devices/2_Running_quantum_circuits_on_QPU_devices.ipynb

```diff
@@ -96,17 +96,17 @@
     "print('\\n') \n",
```

Comment:
<p>Should be Ankaa-2 and Harmony. </p><p> </p>

<pre class="ql-syntax" spellcheck="false">Quantum Task Summary
{'arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3': {'shots': 2000, 'tasks': {'COMPLETED': 2}}, 'arn:aws:braket:us-east-1::device/qpu/ionq/Harmony': {'shots': 1000, 'tasks': {'COMPLETED': 1}}}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>